issues installing

tall"
npm ERR! node v7.0.0
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE

npm ERR! react-native-ble-plx@0.3.3 postinstall: `bash build_ios_frameworks.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-native-ble-plx@0.3.3 postinstall script 'bash build_ios_frameworks.sh'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-native-ble-plx package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     bash build_ios_frameworks.sh
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-native-ble-plx
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-native-ble-plx
npm ERR! There is likely additional logging output above.

Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "--save" "react-native-ble-plx"
npm ERR! node v7.9.0
npm ERR! npm  v4.2.0
npm ERR! code ELIFECYCLE
npm ERR! errno 1

npm ERR! react-native-ble-plx@0.4.0 postinstall: `bash build_ios_frameworks.sh`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-native-ble-plx@0.4.0 postinstall script 'bash build_ios_frameworks.sh'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-native-ble-plx package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     bash build_ios_frameworks.sh
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-native-ble-plx
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-native-ble-plx
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/marwazribi/.npm/_logs/2017-05-05T14_59_43_032Z-debug.log

and i'm a mac user

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "example": "examples"
   },
   "scripts": {
-    "postinstall": "bash build_ios_frameworks.sh",
+    "postinstall": "node ./postinstall.js",
     "test": "jest",
     "lint": "flow && documentation lint index.js",
     "docs": "documentation build index.js -o docs --config documentation.yml -f html"

--- a/postinstall.js
+++ b/postinstall.js
@@ -1,0 +1,8 @@
+var childProcess = require('child_process');
+
+if (process.platform === 'darwin') {
+  childProcess.spawn('./build_ios_frameworks.sh', [], {
+    shell: 'bash',
+    stdio: 'inherit',
+  });
+}

